packages/workshop-app/app/routes/_app+/_exercises+/_layout.tsx

```diff
@@ -24,33 +20,30 @@ export async function loader({ request }: DataFunctionArgs) {
 		getPlaygroundAppName(),
 	])
 
-	const result = json(
-		{
-			workshopTitle,
-			exercises: exercises.map(e => ({
-				exerciseNumber: e.exerciseNumber,
-				title: e.title,
-				solutions: e.solutions.map(({ stepNumber, title, name }) => ({
-					stepNumber,
-					title,
-					name,
-				})),
-				problems: e.problems.map(({ stepNumber, title, name }) => ({
-					stepNumber,
-					title,
-					name,
-				})),
+	const [, match] = playgroundAppName?.match(/exercises\.(\d*)/) ?? []
+	const playground = {
+		appName: playgroundAppName,
+		exerciseNumber: Number(match),
+	}
+
+	const result = json({
+		workshopTitle,
+		exercises: exercises.map(e => ({
+			exerciseNumber: e.exerciseNumber,
+			title: e.title,
+			solutions: e.solutions.map(({ stepNumber, title, name }) => ({
+				stepNumber,
+				title,
+				name,
 			})),
-			playgroundAppName,
-		},
-		{
-			headers: {
-				'Cache-Control': 'public, max-age=300',
-				Vary: 'Cookie',
-				'Server-Timing': getServerTimeHeader(timings),
```

Comment:
We need to keep this header so we can see server timings in the browser.

Reply:
i can remove only `'Cache-Control': 'public, max-age=300',`